### ISSUE TITLE
Redesign the Home page according to the prototype that i will share. 

### ISSUE THREAD

**Issue by usha-madithati** [usha-madithati/esmart.github.io#21]

**Comment by nandu-99** [comment]
Please assign this issue to me

**Comment by usha-madithati** [comment]
Good to see you. Start working accordingly as soon as I send you the design.
Thank You

**Comment by nandu-99** [comment]
Thank you so much

**Comment by SwapnaneelR** [comment]
https://github.com/SwapnaneelR/esmart.github.io.git

**Comment by usha-madithati** [comment]
Are you interested in designing responsive UI for our website?

**Comment by cpratik1307** [comment]
I am thrilled to be participating in GirlScript Summer of Code 2k24 as a passionate computer science student with a keen interest in web development. I am eager to contribute to your project and leverage my skills to make meaningful contributions. I look forward to collaborating with you and the team to create impactful solutions.

**Comment by solanki505** [comment]
I  want to work on this issue please assign me this issue under GSSOC'24

**Comment by usha-madithati** [comment]
Can you redesign the next section of our website according to the prototype
I will share . Is that ok with you?

**Comment by kanikaa-3018** [comment]
please assign this issue to me for GSSoc'24, i will improve the entire layout and also quality of images along with the animations and hover effects of the buttons

**Comment by AtharvaShinde253** [comment]
I can redesign the next section of the website.
Looking forward to collaborating on this exciting project!
Please assign me with this issue!

**Comment by Abdal192003** [comment]
Please assign me this task under gssoc'24, i will try best to solve this issue.
Thanks

**Comment by gitsofaryan** [comment]
hi @usha-madithati can you please assign mew this task
https://www.linkedin.com/in/aryan-jain07/

**Comment by nsk6704** [comment]
Hello @usha-madithati!

Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?

I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!

This is my LinkedIn profile 

https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/

**Comment by Sudiksha18** [comment]
hii @usha-madithati can you assign me this task under gssoc24

**Comment by usha-madithati** [comment]
> Hello @usha-madithati!
> 
> Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?
> 
> I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!
> 
> This is my LinkedIn profile 
> 
> https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/

I want you to design complete ui for our website is that ok for you

**Comment by usha-madithati** [comment]
> hii @usha-madithati can you assign me this task under gssoc24

Heyyy @Sudiksha18 can you design ui for our website

**Comment by nsk6704** [comment]
> > Hello @usha-madithati!
> > Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?
> > I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!
> > This is my LinkedIn profile
> > https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/
> 
> I want you to design complete ui for our website is that ok for you

Yes, will give my best shot, please do send in your requirements

**Comment by usha-madithati** [comment]
> > > Hello @usha-madithati!
> > > Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?
> > > I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!
> > > This is my LinkedIn profile
> > > https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/
> > 
> > I want you to design complete ui for our website is that ok for you
> 
> Yes, will give my best shot, please do send in your requirements

Drop your whatsapp details so that I can contact you

**Comment by nsk6704** [comment]
> > > > Hello @usha-madithati!
> > > > Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?
> > > > I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!
> > > > This is my LinkedIn profile
> > > > https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/
> > > 
> > > 
> > > I want you to design complete ui for our website is that ok for you
> > 
> > 
> > Yes, will give my best shot, please do send in your requirements
> 
> Drop your whatsapp details so that I can contact you

Sure, could I share on LinkedIn?

**Comment by usha-madithati** [comment]
> > > > > Hello @usha-madithati!
> > > > > Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?
> > > > > I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!
> > > > > This is my LinkedIn profile
> > > > > https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/
> > > > 
> > > > 
> > > > I want you to design complete ui for our website is that ok for you
> > > 
> > > 
> > > Yes, will give my best shot, please do send in your requirements
> > 
> > Drop your whatsapp details so that I can contact you
> 
> Sure, could I share on LinkedIn?

ushamadithati712@gmail.com drop your whatsapp contact here

**Comment by nsk6704** [comment]
> > > > > > Hello @usha-madithati!
> > > > > > Could you kindly give me an opportunity to contribute to your Website by assigning this task to me?
> > > > > > I have interned at Codsoft and have experience  in designing responsive websites, besides being certified by freeCodeCamp!
> > > > > > This is my LinkedIn profile
> > > > > > https://www.linkedin.com/in/saketh-kashyap-nagendra-87b317249/
> > > > > 
> > > > > 
> > > > > I want you to design complete ui for our website is that ok for you
> > > > 
> > > > 
> > > > Yes, will give my best shot, please do send in your requirements
> > > 
> > > 
> > > Drop your whatsapp details so that I can contact you
> > 
> > 
> > Sure, could I share on LinkedIn?
> 
> [ushamadithati712@gmail.com](mailto:ushamadithati712@gmail.com) drop your whatsapp contact here

Right, its 7093445470

**Comment by Sudiksha18** [comment]
yes @usha-madithati i can do it can you share protype type and I am good at html ,js,css,php also
linkedlin:https://www.linkedin.com/in/sudiksha-thatipelli

**Comment by Silentman87** [comment]
hello @usha-madithati  i would like to redesign homepage. please assign to me

**Comment by Paavni-Goyal** [comment]
Hello @usha-madithati can you please assign the issue to me, i would like to work on the same.

**Comment by Laksh-01** [comment]
Hello @usha-madithati , can you please assign me with the issue. I am a 2nd year btech greaduate from NSUT. You can also check my linkedin profile https://www.linkedin.com/in/lakshsachdeva/  .  Hopefully i can deliver something which is required and satisfactory from your end.

**Comment by Vikashchou-dhary** [comment]
Hello @usha-madithati i am interested to work on these issue . please assign it to me

**Comment by its-kritika** [comment]
Hello @usha-madithati i am interested to work on this issue. Here's my linkedIn profile : https://www.linkedin.com/in/kritika-kesarwani-034104239/

**Comment by Vaishnavi2445** [comment]
Hello! @usha-madithati 
I'm eager to contribute to your project by enhancing the homepage and implementing advanced features like voice navigation. I envision users being able to hear the content as they navigate through the site. Could you provide me with the opportunity to contribute to GSSOC?

thank You!

**Comment by sohan-gupthak** [comment]
Hello @usha-madithati i would love to work on this, please assign me this issue under GSSoC'24
LinkedIn - https://www.linkedin.com/in/sohan-guptha/

**Comment by kavyachandarkar** [comment]
hello @usha-madithati i would love to work on this, please assign me this issue under GSSoC'24

**Comment by anupa-rb** [comment]
Hi there, I'm a beginner at open-source contribution so I want to gain hands-on experience by contributing to this project through GSSOC'24. Please assign this issue to me. Thank you.

**Comment by sameekshaa19** [comment]
Hello @usha-madithati . I am interested in working on this issue. Please assign this issue to me.

**Comment by 143divya** [comment]
I am interested to working on this issue. I'm  a btech first year student and new to open source . You can see my linkedin profile here   https://www.linkedin.com/in/divyanjali-sharma%F0%9F%8F%85-a4945a293/

**Comment by Sanchayita23104** [comment]
I would love to work on this issue, please assign this task to me

**Comment by Priti9075** [comment]
I'm incredibly enthusiastic about tackling the "Redesign the Home page according to the prototype that i will share." issue for GSoC 2024! My skills in HTML , CSS , JS perfectly align with this project's needs. Is there a possibility of me being assigned this issue?
